nailgun/entities.py

```diff
@@ -4827,6 +4836,144 @@ def upload_content(self, synchronous=True, **kwargs):
             )
         return json
 
+    def content_upload_id(self, synchronous=True, **kwargs):
+        """Create an upload request
+
+        :param synchronous: What should happen if the server returns an HTTP
+            202 (accepted) status code? Wait for the task to complete if
+            ``True``. Immediately return the server's response otherwise.
+        :param kwargs: Arguments to pass to requests.
+        :returns: The server's response, with all JSON decoded.
+        :raises: ``requests.exceptions.HTTPError`` If the server responds with
```

Comment:
Don't you need to explicitly raise this exception by examining the response?

Reply:
@omaciel you are right, it never raises for 4xx or 5xx, `requests` will raise only if address is not found
 `[Errno -5] No address associated with hostname`)

However as all our other methods are not raising I assume we do not want it to raise error but only logging the errors on logger output.

So we need either to remove all those `:raises:` from all methods here or make `nailgun.client` to raise the exceptions or capture explicitly

Reply:
As the guy who usually look into the test results trying to figure out, what went wrong, I vote for raising a proper exception.

Reply:
My expectation was that `_handle_response` does this for me as it calls `raise_for_status`?

Either way, I don't think this change would belong into this PR, right?

Reply:
@evgeni yeah you are right! I missed the `raise_for_status` there https://github.com/SatelliteQE/nailgun/blob/0d186d822941be318bd637285fd0a75d901f097d/nailgun/entities.py#L113 So no change is needed and actually your PR ir right and don't need to raise explicitly here 
c/c @omaciel